locale/en/corteza-webapp-compose/general.yaml

```diff
@@ -178,6 +178,15 @@ variants:
   warning: Warning
 editor:
   unsavedChanges:  Unsaved changes will be lost. Do you wish to leave the page?
+  file:
```

Comment:
If this is gonna be under general I suggest you move this to its own key on the root under webcam instead of file.
It should be 
```
editor:
  unsavedChanges:  Unsaved changes will be lost. Do you wish to leave the page?
webcam:
  title: Capture Photo
  buttons:
    cancel: Cancel
    capture: Capture photo
````

instead